src/vcpkg/packagespec.cpp

```diff
@@ -76,7 +73,7 @@ namespace vcpkg
     {
         if (platform)
         {
-            return {msg::format(msgIllegalPlatformSpec).data(), expected_right_tag};
+            return {msg::format(msg::msgErrorMessage).append(msgIllegalPlatformSpec).data(), expected_right_tag};
```

Comment:
I don't want to mess with it in this change, but I think we need a shorter way to get to `msg::format(msg::msgErrorMessage)`

Reply:
format_error() has my vote.